code/datums/emotes.dm

```diff
@@ -87,11 +87,13 @@
 
 	user.log_message(msg, LOG_EMOTE)
 
+	var/space = should_have_space_before_emote(html_decode(msg)[1]) ? " " : ""
```

Comment:
```suggestion
	var/space = should_have_space_before_emote(html_decode(msg)[1]) ? " " : null
```

Reply:
why? typefucking for no reason makes bad code.

Reply:
because no separator (space) should be no separator not an empty string.
Everything in dm is nullable so I don't get the "typefucking" comment.